ko/lessons/advanced/otp-supervisor.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: OTP 슈퍼바이저
+category: advanced
+order: 5
+lang: ko
+---
+
+슈퍼바이저(Supervisor)는 다른 프로세스를 감시한다는, 단 하나의 목적에 특화된 프로세스입니다. 자식 프로세스가 실패하면 자동으로 재시작해주므로 장애에 대한 내성이 높은(Fault-tolerant) 애플리케이션을 만들 수 있게 해줍니다.
```

Comment:
(Maybe) `다른 프로세스를 감시한다는, 단 하나의 목적에 특화된` -> `오직 다른 프로세스를 감시하는 목적에만 특화된`
(Maybe) `장애에 대한 내성이 높은` -> `무정지`
Because, many erlang documents in korean refer `Fault-tolerance`  as `무정지성`